epoch/api/channels_api_usage.md

```diff
@@ -173,18 +173,26 @@ The initiator receives the following message
  }
 ```
 
+#### Signed channel_create_tx
+Both participans receive the co-signed `channel_create_tx`:
+```
+{'action': 'on_chain_tx',
+ 'payload': {'tx': 'tx$9mmqY8QRoYS8QR5nyRkWwTyhwecE8NzHotRK1yMjvfvyPfsaBNwWFDrmzG3M5rHKARm39AqeJpgN4Aj7V57yP4KusptAvMA2r3y593vr51ubXN8zo8t8qg7Arb32wWhmYDvRXVGKhfYoqu91BpkammeY69xCjGuQrXKDk3t9aPCcrvM3PbERAMvkmvD9dsb2H5iujGin8qZTy42xsUYS6QcTiaomxo8CmieGSvZkAai1KCZ84MZzsAjsR1FP54Su7rfBPXZTd5u2AmyF7LwZoHQsJEspLnzXxun2MtKTkhR3du8PY'
+            }
+}
+```
+Using its hash participants can track its progress on the chain - entering the mempool, block inclusion and a number of confirmations.
+
 #### Transaction in mempool
-At this point both parties had signed the `channel_create_tx` transaction and
-were informed of each other's signing. The transaction is posted by the state
-channel's software to the node and goes to the mempool. One can validate it
-using the HTTP API:
+At this point both parties had received the co-signed the `channel_create_tx` transaction. The transaction is posted by the state
+channel's software to the node and goes to the mempool. Having calculated its hash, one can validate it using the HTTP API:
 ```
 $ curl 'http://localhost:3013/v2/tx/th$hNyHzj4dSzyBqReAMR36GGz1mhuXxQFuES3AnPkXkuY2w6dZb?tx_encoding=json'
 ```
 if the `block_hash` is `none` - then the transaction is still in the mempool.
 
 ### Block inclusion
-When the transaction is included in a block - this is the first confirmation. A block height timer
+When the transaction is included in a block - this is the first confirmation. A block height timer is
 started and it ends after `minimum_depth + 1` confirmations. Default value for
```

Comment:
Strictly speaking, isn't it `minimum_depth` confirmations after the block that includes the `create_tx`?

Reply:
As discussed, this is fine